app/src/main/java/com/rizadwi/snapsift/common/base/BaseFragment.kt

```diff
@@ -0,0 +1,44 @@
+package com.rizadwi.snapsift.common.base
+
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.fragment.app.Fragment
+import androidx.viewbinding.ViewBinding
+
+abstract class BaseFragment<T : ViewBinding> : Fragment() {
+
+    private var _binding: T? = null
```

Comment:
why use _ as variable name?

Reply:
i thought it's a best practice to prefix a private variable using underscore. moreover i expose that variable publicly using kotlin getter with the name `binding` like below

```
val binding: T get() = _binding!!
```